src/style.css

```diff
@@ -0,0 +1,135 @@
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.productCard {
+  width: 198px;
+  display: inline-flex;
+  flex-direction: column;
+
+  margin: 16px;
```

Comment:
it's not clear why card has margins, it shouldn't have them, delete this property